src/Command/IndexCommand.php

```diff
@@ -18,7 +18,7 @@ abstract class IndexCommand extends Command implements ContainerAwareInterface
 
     protected $indexManager;
```

Comment:
What do you think about adding: `final` class, `private` properties, and also the `@internal` annotation to all console commands.

Reply:
done, properties have to stay `protected` tho